krikri.gemspec

```diff
@@ -57,12 +57,12 @@ Gem::Specification.new do |s|
   # properly.
   s.add_dependency "bootstrap-sass", "3.3.4.1"
 
-  s.add_development_dependency 'krikri-spec', '~> 0.0'
-  s.add_development_dependency "sqlite3"
-  s.add_development_dependency "jettywrapper", '~> 2.0'
-  s.add_development_dependency "rspec-rails", '~> 3.3'
-  s.add_development_dependency 'webmock'
-  s.add_development_dependency 'factory_girl_rails', '~>4.4.0'
-  s.add_development_dependency 'pry-rails'
-  s.add_development_dependency 'timecop'
+  s.add_development_dependency "krikri-spec", "~> 0.0"
```

Comment:
Someone else can comment to verify ... @anarchivist ?

I'm being lazy :-) ... Have been googling and haven't found confirmation of this yet.